include/ast/section.h

```diff
@@ -283,5 +283,16 @@ class AOTSection {
   /// @}
 };
 
+class Component;
+
+class ComponentSection : public Section {
+public:
+  /// Getter of content.
+  std::vector<Component *> &getContent() noexcept { return Content; }
+
```

Comment:
The fix cannot be seemed as correct

1. `Span` is not provided here.
2. Now we have to ensure `Component *` didn't get misused.